amazonka/src/Network/AWS/Auth.hs

```diff
@@ -98,10 +99,14 @@ envSecretKey = "AWS_SECRET_ACCESS_KEY"
 envSessionToken :: Text -- ^ AWS_SESSION_TOKEN
 envSessionToken = "AWS_SESSION_TOKEN"
 
--- | Credentials profile environment variable.
+-- | Default credentials profile environment variable.
 envProfile :: Text -- ^ AWS_PROFILE
 envProfile = "AWS_PROFILE"
 
+-- | Default region environment variable
+envRegion :: Text -- ^ AWS_REGION
+envRegion = "AWS_REGION"
```

Comment:
@brendanhay I'm curious where this is documented, our understanding is that `AWS_DEFAULT_REGION` is the official environment variable.

http://docs.aws.amazon.com/cli/latest/topic/config-vars.html#general-options